packages/datadog_flutter_plugin/README.md

```diff
@@ -12,9 +12,9 @@
 
 [//]: # (SDK Table)
 
-iOS SDK | Android SDK | Browser SDK 
-:-----: | :---------: | :---------: 
-1.9.0 | 1.12.0-alpha2 | ❌
+| iOS SDK | Android SDK | Browser SDK |
+| :-----: | :---------: | :---------: 
```

Comment:
shouldn't it be a '|' at the end of this line as well? Like that:

```suggestion
| :-----: | :---------: | :---------: |
```

Reply:
Yeah, it doesn't matter as much as the one at the start but it's a good catch.